### PR TITLE
867. Transpose Matrix

### PR DESCRIPTION
**Pull request by Sauravsingh-Rana**
### Added a new code to the C++ folder.

Write here what you have changed in the codebase

### Issue no.(must) - #1536 

Pr will be close and marked as spam. If issue number not found or issue was assigned to someone else. 
Marking as spam can block your account from HacktoberFest.

### Self Check(Tick After Making pull Request)

- [ *] This issue was assigned to me if not it will be marked as spam.
- [ *] My file is in the proper folder
- [ *] I am following clean code and Documentation.
- [ *] I have added the title and what program will do first in the file
- [ *] I have named the file as (Problem #). (Problem Title(in Camel Case)).(extension)
      ex. 2. AddToNumbers.py

### CODE REVIEW

**Review by Yashica7 — CHANGES_REQUESTED** [review]
Please add a description or comments of what your code does

File: C++/867. TransposeMatrix.cpp [hunk 1]
```diff
@@ -0,0 +1,40 @@
+// Problem - 867. Transpose Matrix
+
```
Comment (Sauravsingh-Rana) [inline C++/867. TransposeMatrix.cpp:2]: `//This Code will help you in Transposing any square matrix.`

**Comment by Sauravsingh-Rana** [comment]
> Please add a description or comments of what your code does

Thank you @Yashica7 for improvising me. Can you please check now the modified file, if there is anything i forget to add it would be helpful to me.

**Comment by Yashica7** [comment]
Looks better, will merge this now

**Comment by Sauravsingh-Rana** [comment]
> Looks better, will merge this now

Thank you... @Yashica7

**Pull request merged**